css/main.css

```diff
@@ -735,15 +735,17 @@ select.error {
 /* Sponsors section*/
 
 .block-sponsor img {
-    opacity: .2;
+    
+    opacity: .18;
     width: 85%;
 }
 .block-sponsor img:hover {
-    opacity: .4;
-    transform: scale(1.05);
-    -webkit-transform: scale(1.05);
-    -ms-transform: scale(1.05);
-    -moz-transform: scale(1.05);
+    padding-top: 25px;
+    opacity: 1.10;
+    transform: scale(1.50);
```

Comment:
The scale ratio is too much, decrease it for a smooth experience. A tiny amount will do the magic.

Reply:
changed the transform scale to 1.10(kept at 1 for emphasis animations)